server/run-tests.sh

```diff
@@ -2,7 +2,7 @@
 
 # Run tests with various configurations:
 
-TEST_COMMAND="cargo test --all --all-features --all-targets"
+TEST_COMMAND="cargo test --all --all-features --all-targets -- --test-threads=1"
```

Comment:
I could have sworn we had this in already. 🤔

Reply:
It was added, then removed in later commit.

Reply:
Ah cool.